### PR TITLE
DC-3310 - update eis payload

### PR DESCRIPTION
**Pull request by elabeca**
testing EIS integration in QA - this code is subject to change

### CODE REVIEW

File: app/uk/gov/hmrc/securemessage/models/QueryMessageRequest.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.securemessage.models
+
+import org.joda.time.DateTime
+import play.api.libs.json.{ Format, JodaReads, JodaWrites, Json, OWrites, Reads, Writes }
+
+/*
```
Comment (OlegEfrem) [inline app/uk/gov/hmrc/securemessage/models/QueryMessageRequest.scala:22]: can you move it to test/resources .json file and reference the link here ?

File: app/uk/gov/hmrc/securemessage/services/Auditing.scala [hunk 2]
```diff
@@ -119,15 +119,15 @@ trait Auditing {
...
     implicit hc: HeaderCarrier,
     ec: ExecutionContext): Unit = {
     val detail = Map(
       txnName           -> "Message forwarded to caseworker",
       "eisResponseCode" -> eisResponseCode.toString,
-      "conversationId"  -> qrw.queryResponse.conversationId,
-      "x-request-id"    -> qrw.queryResponse.id,
-      "message"         -> qrw.queryResponse.message
+      "conversationId"  -> qrw.requestDetail.conversationId,
+      "x-request-id"    -> qrw.requestDetail.id,
```
Comment (OlegEfrem) [inline app/uk/gov/hmrc/securemessage/services/Auditing.scala:129]: do we need a seetionId as well ?

File: app/uk/gov/hmrc/securemessage/connectors/EISConnector.scala [hunk 3]
```diff
@@ -49,30 +47,29 @@ class EISConnector @Inject()(
...
         s"$eisBaseUrl$eisEndpoint",
-        queryResponse,
+        queryMessageRequest,
         Seq(
           (CONTENT_TYPE, MimeTypes.JSON),
           (ACCEPT, MimeTypes.JSON),
           (AUTHORIZATION, s"Bearer $eisBearerToken"),
           (DATE, DateTime.now().toString("EEE, dd MMM yyyy HH:mm:ss z")),
-          (CustomHeaders.CorrelationId, UUID.randomUUID().toString),
+          (CustomHeaders.CorrelationId, queryMessageRequest.requestCommon.acknowledgementReference),
```
Comment (OlegEfrem) [inline app/uk/gov/hmrc/securemessage/connectors/EISConnector.scala:61]: do we need request and session ids as well ?

**Review by OlegEfrem — DISMISSED** [review]

File: app/uk/gov/hmrc/securemessage/connectors/EISConnector.scala (hunk 3 above)
Comment (elabeca) [reply app/uk/gov/hmrc/securemessage/connectors/EISConnector.scala:61]: no

File: app/uk/gov/hmrc/securemessage/services/Auditing.scala (hunk 2 above)
Comment (elabeca) [reply app/uk/gov/hmrc/securemessage/services/Auditing.scala:129]: not sure, it's not the focus of my story

File: app/uk/gov/hmrc/securemessage/models/QueryMessageRequest.scala (hunk 1 above)
Comment (elabeca) [reply app/uk/gov/hmrc/securemessage/models/QueryMessageRequest.scala:22]: meant to delete

Comment (elabeca) [reply app/uk/gov/hmrc/securemessage/models/QueryMessageRequest.scala:22]: oh yes will add to test json resources

**Review by OlegEfrem — APPROVED** [review]

**Pull request merged**